.github/workflows/code-style.yml

```diff
@@ -14,10 +14,10 @@ jobs:
         autopep8 --recursive --in-place --aggressive --aggressive .
         black .
     - name: Commit and push changes
-      uses: stefanzweifel/git-auto-commit-action@v2.0.0
+      uses: stefanzweifel/git-auto-commit-action@v4
       with:
         commit_author_email: ''
```

Comment:
Please remove commit_author_email and commit_author_name

Reply:
ok

Reply:
Updated the changes ! please verify now @vfdev-5